scenario/consistency_checker.py

```diff
@@ -190,6 +190,12 @@ def _check_workload_event(
             f"workload event should start with container name. {event.name} does "
             f"not start with {event.container.name}.",
         )
+    else:
+        command_prefixes = [exec.command_prefix for exec in event.container.execs]
+        if len(command_prefixes) != len(set(command_prefixes)):
+            errors.append(
+                f"container {event.container.name} has multiple execs with the same command prefix.",
+            )
```

Comment:
I'd use a `collections.Counter` as that allows interpolation of the offending key into the error message.

Reply:
This is done elsewhere in the consistency checker, so I guess it would not have any harm and would be a little nicer. In general, I'd rather avoid an extra import just for information that the caller probably trivially has (the test is unlikely to be long and almost certainly has an obvious single container with a small number of execs).